ttnn/cpp/ttnn/operations/pool/maxpool/device/max_pool2d_multi_core_program_factory.cpp

```diff
@@ -137,11 +137,14 @@ MaxPoolNew::MultiCore::cached_program_t max_pool_2d_multi_core_sharded_with_halo
             .set_globally_allocated_address(*reader_indices_buffer);
     auto in_reader_indices_cb = tt::tt_metal::CreateCircularBuffer(program, all_cores, in_reader_indices_cb_config);
 
+    auto in_cb_sz = (input_shape[3] * kernel_size_hw_padded) > (conv_op_utils::constants::TILE_HW * 8)
+                        ? (conv_op_utils::constants::TILE_HW * 8)
+                        : input_shape[3] * kernel_size_hw_padded;
```

Comment:
what is `8` here?

Reply:
Maximum number of tiles which can be reduced at a time. Updated with proper name and comment for better understaning.